integration-tests/mysql-client-tests/r/rmariadb-test.r

```diff
@@ -0,0 +1,65 @@
+library(RMariaDB)
+library(DBI)
+
+args = commandArgs(trailingOnly=TRUE)
+
+user = args[1]
+port = args[2]
+db = args[3]
+
+conn = dbConnect(RMariaDB::MariaDB(), host="127.0.0.1", port = port,
+                 username = user, dbname = db)
+
+# check standard queries
+queries = list("create table test (pk int, value int, primary key(pk))",
+               "describe test",
+               "insert into test (pk, value) values (0,0)",
+               "select * from test")
+
+responses = list(NULL,
+                 data.frame(Field = c("pk", "value"), Type = c("int", "int"), Null = c("NO", "YES"), Key = c("PRI", ""), Default = c("", ""), Extra = c("", "")),
```

Comment:
As discussed offline, I guess we need `stringsAsFactors = FALSE` as parameters to these data.frame calls if we're going to run on older R versions and we want the data frames coming back from the database layer to look as similar as possible to the data frames we are creating here.